optuna/trial/_trial.py

```diff
@@ -469,6 +469,10 @@ def objective(trial):
         Raises:
             :exc:`NotImplementedError`:
                 If trial is being used for multi-objective optimization.
+            :exe:`ValueError`:
+                If ``step`` is negative.
+            :exe:`TypeError`:
+                If the type of ``value`` is not obj:`float`.
```

Comment:
Ah, I'm sorry but I missed `:`. The current rendering result is as follows:

![image](https://user-images.githubusercontent.com/3255979/143733184-99279c4f-1e0a-484d-a40d-c78570673e01.png)

```suggestion
                If the type of ``value`` is not :obj:`float`.
```

Reply:
Thank you, permission to modify.